plugins/content_distribution/providers/tvinci/lib/admin/TvinciProfileConfiguration.php

```diff
@@ -83,8 +83,18 @@ protected function layoutForm($order)
 				'filters'		=> array('StringTrim'),
 		));
 
+		$this->addElement('text', 'schema_id', array(
+				'label'			=> 'Schema ID:',
+				'filters'		=> array('StringTrim'),
+		));
+
+		$this->addElement('text', 'language', array(
```

Comment:
Partially done since I lack meaning to the values - will wait for answer from Arik for the creation of the enum